nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -204,6 +204,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   `globalRedirect` can now have redirect codes other than 301 through
   `redirectCode`.
 
+- `libjxl` 0.9.0 dropped support for the butteraugli API. You will no longer be able to set `enableButteraugli` on `libaom`.
```

Comment:
I think it would be useful to add a link of some sort to this line. Maybe something like this?

```suggestion
- `libjxl` 0.9.0 [dropped support for the butteraugli API](https://github.com/libjxl/libjxl/pull/2576). You will no longer be able to set `enableButteraugli` on `libaom`.
```